lib/t/feeds.ex

```diff
@@ -143,13 +146,24 @@ defmodule T.Feeds do
   end
 
   defp empty_feeded_profiles(user_id) do
+    primary_rpc(__MODULE__, :local_empty_feeded_profiles, [user_id])
```

Comment:
Otherwise we pay for the writes in terms of latency here and ...

Reply:
but it's still much better than having one node (because of intra-AWS connectivity), right?

Reply:
no, latencies are almost the same for public/private networking